README.md

```diff
@@ -186,7 +186,7 @@ More details are available [here](https://github.com/ceph/ceph-csi/issues/463)
 
 ## Dev standup
 
-A regular dev standup takes place every [Tuesday at
+A regular dev standup takes place every [Monday at
 12:00 PM UTC](https://meet.google.com/vit-qdhw-nyh) Convert to your local
```

Comment:
Can you check the time and confirm again?

Reply:
Yes, of course I did when i posted the pr.